RELEASE_NOTES.md

```diff
@@ -10,4 +10,5 @@
         * [SER-1891] - Implement SER-1605 user currencies, AccountNo/Shop and AMS restrictions
         * [SER-1971] - Setup azure pipeline
         * [SER-1597] - Use the required date time format when filtering transactions by date
-        * [SER-1926] - Fix errors that occur when exporting transactions as csv
\ No newline at end of file
+        * [SER-1926] - Fix errors that occur when exporting transactions as csv
+        * [SER-1596] - Filter incoming requests to pay by external Id
```

Comment:
I think this will be released in a new version. Please add an entry for a new version number (probably 1.1.0) at the top for this feature.

Other than that, code looks good to me.

Reply:
Also, since this looks like the only feature that'd get released in the new version, please update the version number in the [package.json](https://github.com/one-acre-fund/oaf-ph-ee-operations-web/blob/a40f4a70a27c73fad4f8036df76c3ea59d3fce66/package.json#L3) file

Reply:
From @Vikeln 
"There's no need to add it to the release notes since there was an initial PR for it and it was already listed in the release notes. this was just a fix"

Reply:
I thought this was on the backend, sorry @amydoxym lemme do it now